gcsweb/cmd/gcsweb/gcsweb.go

```diff
@@ -330,8 +372,28 @@ func otherRequest(w http.ResponseWriter, r *http.Request) {
 	http.NotFound(w, r)
 }
 
+// bucketAlias permits a naive URL rewriting functionality.
+type bucketAlias struct {
```

Comment:
Seems like you could use `map[string]string` here (from -> to) instead of an array of mappings? Any reason not to?

Reply:
No, I don't have any particular reason other than iterating a map is a non deterministic operation and that behavior gives me problems from time to time. I guess it won't make any difference here, so I will modify it.